src/Compilers/CSharp/Test/Symbol/Symbols/SymbolErrorTests.cs

```diff
@@ -13950,31 +13912,6 @@ public void CS0764ERR_PartialMethodUnsafeDifference()
                 Diagnostic(ErrorCode.ERR_PartialMethodUnsafeDifference, "M2").WithLocation(5, 25));
         }
 
-        [Fact]
-        public void CS0766ERR_PartialMethodMustReturnVoid()
```

Comment:
>CS0766ERR_PartialMethodMustReturnVoid [](start = 20, length = 37)

Please keep the test even though behavior has changed.

Reply:
Restored the test.